python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -877,9 +877,17 @@ def info(self, full=False):
                 s += ", %i other concurrent sessions:\n * %s" % (len(other_sessions), "\n * ".join(other_sessions))
         return s
 
+    def other_session_active(self):
+        """ Returns a True/False for if other sessions have been detected from the Repository """
+        if self.repository.get_other_sessions():
+            return True
+        else:
+            return False
+
     def print_other_sessions(self):
-        other_sessions = self.repository.get_other_sessions()
-        if len(other_sessions) > 0:
+        """ Prints a warning if other active sessions have been detected """
+        if self.other_session_active():
+            other_sessions = self.repository.get_other_sessions()
```

Comment:
If you rally want to hide the contents of the session list then instead how about adding a method to `GangaRepository`:
```python
def has_other_sessions(self):
    """Are there other running sessions"""
    return bool(self.get_other_sessions())
```
It makes much more sense being in the repository than in the registry. This function can then be used anywhere you want a boolean result.